### ISSUE TITLE
Error on running enas

### ISSUE THREAD

**Issue by manhquang144** [melodyguan/enas#67]
I am getting error when trying to run the enas with CIFAR10. 
The error is:
**Failed to get device properties, error code: 30**

Is that a problem with my GPU? 
Could anyone can help me to fix that? Thank you so much

**Error Log**
2018-09-23 12:12:57.280583: I T:\src\github\tensorflow\tensorflow\core\common_runtime\gpu\gpu_device.cc:1484] Adding visible gpu devices: 0
2018-09-23 12:13:19.540615: I T:\src\github\tensorflow\tensorflow\core\common_runtime\gpu\gpu_device.cc:965] Device interconnect StreamExecutor with strength 1 edge matrix:
2018-09-23 12:13:19.540864: I T:\src\github\tensorflow\tensorflow\core\common_runtime\gpu\gpu_device.cc:971]      0 
2018-09-23 12:13:19.541034: I T:\src\github\tensorflow\tensorflow\core\common_runtime\gpu\gpu_device.cc:984] 0:   N 
2018-09-23 12:13:19.558336: I T:\src\github\tensorflow\tensorflow\core\common_runtime\gpu\gpu_device.cc:1097] Created TensorFlow device (/job:localhost/replica:0/task:0/device:GPU:0 with 1412 MB memory) -> physical GPU (device: 0, name: GeForce 845M, pci bus id: 0000:01:00.0, compute capability: 5.0)
2018-09-23 12:13:44.150772: E T:\src\github\tensorflow\tensorflow\core\grappler\clusters\utils.cc:82] Failed to get device properties, error code: 30
2018-09-23 12:13:45.258997: E T:\src\github\tensorflow\tensorflow\core\grappler\clusters\utils.cc:82] Failed to get device properties, error code: 30
2018-09-23 12:13:46.529868: E T:\src\github\tensorflow\tensorflow\core\grappler\clusters\utils.cc:82] Failed to get device properties, error code: 30
2018-09-23 12:13:47.397507: E T:\src\github\tensorflow\tensorflow\core\grappler\clusters\utils.cc:82] Failed to get device properties, error code: 30
2018-09-23 12:13:48.358465: E T:\src\github\tensorflow\tensorflow\core\grappler\clusters\utils.cc:82] Failed to get device properties, error code: 30
2018-09-23 12:13:49.161798: E T:\src\github\tensorflow\tensorflow\core\grappler\clusters\utils.cc:82] Failed to get device properties, error code: 30
2018-09-23 12:15:54.081286: E T:\src\github\tensorflow\tensorflow\stream_executor\stream.cc:318] Error recording event in stream: error recording CUDA event on stream 000001A6226960F0: CUDA_ERROR_UNKNOWN; not marking stream as bad, as the Event object may be at fault. Monitor for further errors.
2018-09-23 12:15:54.107639: E T:\src\github\tensorflow\tensorflow\stream_executor\cuda\cuda_event.cc:48] Error polling for event status: failed to query event: CUDA_ERROR_UNKNOWN
2018-09-23 12:15:54.108491: F T:\src\github\tensorflow\tensorflow\core\common_runtime\gpu\gpu_event_mgr.cc:206] Unexpected Event status: 1

**Comment by Mistobaan** [comment]
Yes bad GPU configuration. try to run nvidia-smi and make sure that works.
